themes/beaver/layouts/partials/img/homepage.html

```diff
@@ -0,0 +1,3 @@
+{{ $image := .image }}
+{{ $title := .title }}
+{{ partial "img/generic.html" (dict "image" $image "title" $title "widths" (slice 300 730) "sizes" "(max-width: 730px) 100vw, 730px" "attributes" "fetchpriority=\"high\"") }}
```

Comment:
Need to resize image per specific sizes.

And there is observeration that this container is 360px always for all layouts.